config/settings.js

```diff
@@ -27,11 +27,20 @@
  */
 
 window.spinnakerSettings = {
+  defaultProviders: ['aws', 'gce', 'azure', 'cf', 'kubernetes'],
+  feedbackUrl: process.env.FEEDBACK_URL,
   gateUrl: gateUrl,
-  bakeryDetailUrl: bakeryBaseUrl + '/api/v1/global/logs/{{context.status.id}}?html=true',
+  bakeryDetailUrl: bakeryBaseUrl + '/api/v1/global/logs/{{context.status.id}}?html=true'
+  authEndpoint: process.env.AUTH_ENDPOINT,
   pollSchedule: 30000,
-  defaultTimeZone: 'America/New_York', // see http://momentjs.com/timezone/docs/#/data-utilities/
+  defaultTimeZone: process.env.TIMEZONE || 'America/Los_Angeles', // see http://momentjs.com/timezone/docs/#/data-utilities/
   providers: {
+    azure: {
+      defaults: {
+        account: 'azure-test',
```

Comment:
@ewiseblatt Can you please point @AMeng in the right direction as to what (if anything) would need to be modified in the checked-in 'centralized config' stuff to enable the additional accounts to be rewritten here?